### ISSUE TITLE
Add Ombi support

### ISSUE THREAD

**Issue by Ornias1993** [Ornias1993/jailman#17]
It would be nice having support for recent version of Ombi within FreeNAS jails.

This PR has two options/stages:
1. Adding support for older (pro dot-net) ombi jails
2. Getting the current version in (blocked by lack of dot net support)

I don't think it's worth spending time on  getting heavily EOL amateur software running in jails and the security risks are too high imho

**Comment by uofirob** [comment]
For the time being (I know it doesn't fall under this script's wheelhouse), load up a VM with Ubuntu and the install is easy.

**Comment by Ornias1993** [comment]
Yeah it's easy, but for Jailman's sake it would be nice if we could just add it.

I've worked on DotNet support already, IX is going to fix one of the blocking bugs in 11.3U4 and TrueNAS 12 should in theory be compatbile with the beta builds of DotNet on FreeBSD i've gotten my hands on...

It's only a mater of time :)
